src/components/widget/legend/component.js

```diff
@@ -13,14 +13,22 @@ const Legend = ({ groups, direction }) => (
           {groups[g].map(item => (
             <li
               key={`item-${item.color}`}
-              className={styles.widget_legend_list_item}
+              className={classnames(
+                styles.widget_legend_list_item,
+                { [styles.net]: true }
+              )}
             >
               <svg height="12" width="12">
-                <rect width="13" height="13" fill={item.color} />
+                <rect
```

Comment:
The border radius can be achieved with styles on the HTML container. I think is way to opinionated to test for 'net result' here and should be avoided. Add style variants or a classname.